### ISSUE TITLE
Entry manager broke member invite links and uniqs for duplicated teams

### ISSUE THREAD

**Issue by dretelny** [StanfordHCI/foundry#257]
When the team is duplicated, the urls and uniqs in the member popovers show the info from the original team (i.e. the team that was duplicated). I had fixed this and had it working when I resolved the duplicating issues in PR #233 but then when the "entry manager" was added (I think when the folders were introduced) it broke my fixes. If you look at the flashTeamsJSON in the console, it looks like my fixes are still there but now the member data is apparently stored in "_member_data" and "members" -- which include different information for duplicated team. Specifically, the "_member_data" has the incorrect information and the "members" data has the correct info. See screenshot below.

I tried fixing this but couldn't get it to work. @ofagbemi is this something you can fix? You might need to update the createDupTeamStatus function i created in the flash teams controller. Also, is it possible to only save the member info in one place? otherwise this will be extremely difficult to maintain...

![image](https://cloud.githubusercontent.com/assets/5275384/6343904/5dd52452-bba5-11e4-9de8-d0b8f85744e3.png)

**Comment by ofagbemi** [comment]
Fixed <https://github.com/StanfordHCI/foundry/pull/249>. I removed every
reference to the "members" field in flashTeamsJSON. I'm leaving it as an
empty array for right now, but every member is now stored in the
EntryManager. This seems to have resolved the issue with uniques.

Let me know if you find anything!

**Comment by ofagbemi** [comment]
Do the members duplicate properly? In other words, if you duplicate a team do the member links in the duplicated team update (e.g. What I had fixed) or are they the same as the original team? Also, what happens now in previous places we refer to the flash teams member array? We use it in different places of foundry such as the hiring portal so i worry that you removed it?

**Comment by ofagbemi** [comment]
Sorry, I wasn't super clear. I removed all external references (external as
in outside of the EntryManager) and replaced them with calls to
EntryManager functions. The EntryManager uses the flashTeamsJSON "members"
field and is populated by the JSON in its constructor.

The flashTeamsJSON.members field still exists. Now only the EntryManager
and the ruby code on the backend actually touch it. Every interaction with
it on the front-end happens via the EntryManager.

The links in the duplicated team are different. I replaced all the calls in
the hiring portal and other places with calls to EntryManager functions.
